projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/banking/userIdentificationPage.ts

```diff
@@ -27,4 +27,6 @@ export function selectUserIdentification(identification) {
   cy.wait(`@${cartContent}`).then(({ response }) => {
     expect(response.statusCode).to.eq(200);
   });
+  cy.get('.primary-button').should('contain.text', 'Continue');
```

Comment:
This seems redundant since it's also checked in the expression below.

Reply:
Fixed in line below